src/pages/Login/components/EmailVerification.jsx

```diff
@@ -0,0 +1,130 @@
+import React, { useState, useEffect } from 'react';
+import LoginForm from './LoginForm';
+import SignupForm from './SignupForm';
+import styled from 'styled-components';
+import theme from '../../../styles/theme';
+import { fontMix, flexSort, boxSize } from '../../../styles/mixin';
+import emailValidationData from './emailValidation.json';
+
+const EmailVerification = ({ isEmailVerified, setIsEmailVerified }) => {
+  const [email, setEmail] = useState('');
+
+  const handleEmailVerification = () => {
+    const isVerified = emailValidationData.message === true;
```

Comment:
```suggestion
    const isVerified = emailValidationData.message;
```
해당 값 자체가 boolean 값이면 비교하지 않아도 될 것 같습니다

Reply:
다른 부분도 값이 boolean 값인데 비교를 해둔 부분이 있다면 수정해주세요